src/Symfony/Component/Notifier/Bridge/GoogleChat/GoogleChatOptions.php

```diff
@@ -62,6 +62,8 @@ public function toArray(): array
     }
 
     /**
+     * @deprecated google chat cards-v1 deprecated, use "cardV2()" instead.
```

Comment:
`@deprecated since Symfony 6.3, use "cardV2()" instead.`

+ please add a corresponding call to trigger_deprecation in the body of the method

(and see also fabbot failures)

Reply:
Done ✅